pkg/utils/metrics/metrics.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package metrics
```

Comment:
could you please move to the test/utils folder to split it from production code?

Reply:
My bad. Done